docs/tutorial/recent-documents.md

```diff
@@ -39,6 +39,30 @@ will be started with the path of the file added as a command line argument.
 
 ## macOS Notes
 
+### Adding the Recent Documents list to the application menu:
+
+![macOS Recent Documents menu item][menu-item-image]
+
+You can add menu items to access and clear recent documents by adding the following code snippet to your menu's template.
+
+```javascript
```

Comment:
@bhutraraghav this should be `json`

Reply:
@codebytere They were probably added by @erickzhao .